docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.7'
+services: 
+    planner:
+        build: .
+        image: planner:latest
+        ports:
+            - '3001:3001'
```

Comment:
Are you sure this port has to be accessible externally?

Reply:
https://stackoverflow.com/questions/40801772/what-is-the-difference-between-docker-compose-ports-vs-expose

According to this answer -> yes I do.
If I won't use `ports` directive I wouldn't be able to connect to container from host machine.